admin/users.sql

```diff
@@ -127,7 +124,7 @@ CREATE USER IF NOT EXISTS RECOVER_SERVICE
     MUST_CHANGE_PASSWORD = TRUE;
 
 CREATE USER IF NOT EXISTS ADMIN_SERVICE
-    TYPE = SERVICE;
+    TYPE = SERVICE; --noqa: LT02, PRS
```

Comment:
Do we need these rule exceptions? The indent is already a multiple of 4 (LT02) and (PRS) doesn't seem to exist: https://docs.sqlfluff.com/en/stable/reference/rules.html#rule-layout.indent

Reply:
Ohh, PRS is unparseable section. Is it just me, or has sqlfluff been incredibly buggy? It even considers `DROP DATABASE ROLE` statements unparseable, even with dialect set to Snowflake.

Reply:
Ya... I think `sqlfluff` is definitely buggy.. I think it just takes time to add features.